scripts/tasks/svgs-generate.js

```diff
@@ -13,7 +13,7 @@ function svgSymbolCleanUp(shape, sprite, callback) {
 }
 
 function svgAddPolyfill(svgPolyfill, svg) {
-    return svg.replace('</svg>', '<script xmlns="http://www.w3.org/2000/svg" type="text/ecmascript">' + svgPolyfill + '</script></svg>');
+    return svgPolyfill.replace('#SVG#', svg.replace(/(\r\n|\n|\r)/gm,""));
```

Comment:
Removes line breaks and puts SVG content into JS (replacing `#SVG#` placeholder)